protected_app_signals_cost.md

```diff
@@ -0,0 +1,163 @@
+PAS Cost Estimate Explainer
+
+**Authors:** 
+<br> [Stefano Galarraga][0], Google Privacy Sandbox 
+
+
+Overview and Goals
+==================
+
+[Protected App Signals][1] is a new design aimed at the app install ads market. The system is based on server-side execution via the B&A services and [TEE K/V services][2]. Shifting the auction execution on the server side has benefits in terms of latency and computing power but introduces cost to adtechs to run the cloud instances where B&A will reside.
+
+This document is based on the existing documentation about cost analysis and reduction ([B&A][3], K/V) and aims at providing an analysis of how to apply those concepts to the PAS architecture.
+
+Assumptions
+===========
+
+The following assumptions are made in computing the cost:
+
+*   About the K/V server used by the SFE we will consider two cases
+    *   SFE is using a BYOS K/V server
+*   Buyside and Sellside B&A services can be running in different networks
+    *   This means that there could be network cost for communications among the trusted components
+
+Cost Analysis
+=============
+
+The following diagram shows the cost items for the case of BYOS K/V Server
+
+![Diagram for PAS Servers Cost Analysis](images/pas-cost-elements-diagram.png "Diagram for PAS Server Cost Analysis")
+
+With reference to the diagram above there are 9 factors contributing to the cost of running PAS:
+
+*   _A,B_: Cost related to increase in payload sent to and received from the device
```

Comment:
Can we make this sent to and received from the server? Or alternatively, organize it so that the order A,B makes sense (cause we are saying A,B : sent to and received from device, which would be B,A).

Reply:
good point!